content/docs/04.6-vm-management.md

```diff
@@ -33,57 +31,27 @@ Palette VM Orchestrator is particularly suitable in the following scenarios:
 
 - Edge locations that have only a few VMs running, and a hypervisor is no longer desired.
 
-# Prerequisites
```

Comment:
Why did all that section move to a different page? This page is the place where it should be.

Reply:
Discussed with @cloudmaniac  on a call. The reason content is moved is that with the new VMO pack, it no longer makes sense to have the Spectro VM Dashboard doc. The bullet list of tabs within the **Virtual Machines** tab now makes more sense in the Standard VM Operations doc.